src/Nethermind/Nethermind.Serialization.Rlp/TxDecoder.cs

```diff
@@ -544,12 +544,7 @@ private static void EncodeDepositTxPayloadWithoutPayload(T item, RlpStream strea
             bool allowUnsigned = (rlpBehaviors & RlpBehaviors.AllowUnsigned) == RlpBehaviors.AllowUnsigned;
             bool isSignatureOk = true;
             string signatureError = null;
-            if (rBytes == null || sBytes == null)
```

Comment:
Span's cannot be null so this is a cast to null and a length check == 0; calling method can't pass in null either